docs/src/SelectList.doc.js

```diff
@@ -1,16 +1,37 @@
 // @flow strict
 import React, { type Node } from 'react';
-import Example from './components/Example.js';
 import PropTable from './components/PropTable.js';
 import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+import FeedbackCallout from './components/FeedbackCallout.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
+card(
+  <FeedbackCallout link="https://docs.google.com/forms/d/e/1FAIpQLSe7h8kVcD7QqvPvjkE8s8WvnuFfhYvAEQ6L7tZwPgHjJPAbSw/viewform?usp=pp_url&entry.847151274=SelectList" />,
+);
+
 card(
   <PageHeader
     name="SelectList"
-    description={`Use a \`SelectList\` when you have four or more items you want a user to choose from.`}
+    description={`
+    SelectList displays a list of actions or options using the browser’s native select.`}
+    defaultCode={`
+      <SelectList
+        id="selectlistexample"
+        onChange={() => {}}
+        options={[
+          {label:'Alergia', value: 'alergia'},
```

Comment:
typo: Algeria (here and throughout, for both label and value

Reply:
Nice catch - fixed!